_source/_posts/2021-10-20-intro-security-txt.adoc

```diff
@@ -0,0 +1,90 @@
+---
+layout: blog_post
+title: "Security.txt: Make Vulnerabilities Easier to Report"
+author: brian-demers
+by: advocate
+communities: [security]
+description: "Reporting a vulnerability requires special handling, each company has a different process, a security.txt file can help!"
+tags: [securitytxt, security, vulnerabilities, github]
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness
+---
+
+We all know that all software has bugs and that security is hard, but somehow we are still surprised when we see new vulnerabilities.
+
+In 2020, there were 18,395 vulnerabilities reported, which means about 50 new vulnerabilities are reported every day. These numbers only include what has been reported to MITRE's Common Vulnerability and Exposures (CVE) database, which means the actual value is likely much higher. Every year we add more software to the world, so these numbers will only increase. I didn't list this number here to scare you; more so, they are a fact of life, and we need to be aware of how to deal with them.
```

Comment:
```suggestion
In 2020, there were 18,395 vulnerabilities reported, which means about 50 new vulnerabilities are reported every day. These numbers only include what has been reported to MITRE's Common Vulnerability and Exposures (CVE) database, which means the actual value is likely much higher. Every year we add more software to the world, so these numbers will only increase. I didn't list this number here to scare you; more so, they are a fact of life, and we all need to be aware of how to deal with them.
```
added for emphasis. also - it might make sense here to add some more context for readers who don't know much about sec. Like a one-line definition here of what a vulnerability means (as distinguished from a bug), and an even briefer description of MITRE's CVE (the definitive source of blah blah, from the non-profit MITRE which does blah blah)

Reply:
Done!